sympy/integrals/manualintegrate.py

```diff
@@ -541,6 +548,7 @@ def make_second_step(steps, integrand):
         if (constant != 1) and rule:
             rule = ConstantTimesRule(constant, integrand, rule,
                                      constant * integrand, symbol)
+        cachekey = u.xreplace({symbol: _cache_dummy})
```

Comment:
Where is this used?

Reply:
I've no idea how this line got there. :(